streaming/src/java/org/apache/hive/streaming/transaction/InvalidTransactionState.java

```diff
@@ -16,8 +16,11 @@
  * limitations under the License.
  */
 
-package org.apache.hive.streaming;
+package org.apache.hive.streaming.transaction;
```

Comment:
this will break backward compat as these exceptions may be caught clients like Nifi